content/blog/2020/11/2020-11-08-help-test-jenkins.adoc

```diff
@@ -0,0 +1,113 @@
+
+image:/images/post-images/2020-11-08-help-test-jenkins/ui-using-tables.png[Before the pull request]
+
+After the conversion, the **"Manage Jenkins"** page now looks like this:
+
+image:/images/post-images/2020-11-08-help-test-jenkins/ui-using-divs.png[After the pull request]
+
+The user interface improvements from the transition are a nice step forward for Jenkins.
+However, because the user interface improvements require changes in plugins, we need your help.
+
+We need users to test the latest weekly Jenkins releases with the plugins and configurations that are most important to them.
+When users detect an issue, we need them to link:/participate/report-issue/[report the issue] with enough detail that a plugin maintainer can fix the issue.
+Please add the `tables-to-divs-regression` label to the issues.
+The `tables-to-divs-regression` label makes it easier to find issues related to the tables to divs transition.
+
+=== Plugin developers
+
+Several plugins have already been identified that may need changes.
+See the link:https://issues.jenkins-ci.org/browse/JENKINS-62437[Jira epic] for plugins that are likely to need changes for the new user interface layout.
+The list of link:https://issues.jenkins-ci.org/issues/?filter=22840[open `tables-to-divs-regression` issues] can also be used to see plugins that need changes.
+
+If you can assist with plugin testing and code changes, select one of the plugins from that epic, test it, and propose a pull request to help with this user interface transition.
+If you're not comfortable proposing a pull request, describe the problems you see in a bug report.
+
+A link:/doc/developer/views/table-to-div-migration/[tables to divs migration guide] is available.
+It describes areas that typically need to be changed as part of the migration from tables to divs.
+It also includes detailed examples that allow the plugin to continue supporting older Jenkins versions with table layouts and use div layouts for newer Jenkins versions.
+
+== Core - Spring Security replaces Acegi Security
+
+The Jenkins 2.266 release on November 9, 2020 will include the migration to the Spring Security libraries from the Acegi security libraries that link:https://github.com/jglick[Jesse Glick] has proposed and developed through jep:227[Jenkins Enhancement Proposal 227].
+
+This upgrade replaces the Acegi Security library with the current release of the Spring Security library.
+Details of the change are described in jep:227[] and in the link:https://github.com/jenkinsci/jenkins/pull/4848[pull request].
+
+We need users to test the latest Jenkins weekly releases with their plugins and watch for issues related to authentication.
+
+Refer to Jesse Glick's blog post that introduces the details of the change and provides links to the link:https://github.com/jenkinsci/jep/blob/master/jep/227/compatibility.adoc[Spring Security compatibility] table.
```

Comment:
Not released yet. Are we blocked by it. Maybe just add "coming soon"?

Reply:
I merged Jesse's blog post so that it would be visible with tomorrow's release of Jenkins 2.266.  We can merge this with the release of 2.266 or wait one day and then merge it.  Either is great with me.